pkg/controller/nodeipam/config/types.go

```diff
@@ -22,6 +22,10 @@ type NodeIPAMControllerConfiguration struct {
 	ServiceCIDR string
 	// secondaryServiceCIDR is CIDR Range for Services in cluster. This is used in dual stack clusters. SecondaryServiceCIDR must be of different IP family than ServiceCIDR
 	SecondaryServiceCIDR string
-	// NodeCIDRMaskSize is the mask size for node cidr in cluster.
+	// NodeCIDRMaskSize is the mask size for node cidr in single-stack cluster.
```

Comment:
Document how these all interact - are they compatible, mutually exclusive, or does the older form take precedence?

Reply:
@thockin I've documented that they are mutually exclusive in the PR description. But will add those as comments in the follow up.